src/DotNet.Status.Web/GitHubUserTokenHandler.cs

```diff
@@ -88,6 +102,35 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
             return AuthenticateResult.Success(ticket);
         }
 
+        private string ParseBasicAuth(string authParameter)
+        {
+            string value = null;
+            byte[] rentedBytes = _bytePool.Rent(1000);
+            if (Convert.TryFromBase64String(authParameter, rentedBytes, out int size))
+            {
+                Span<byte> basicBytes = rentedBytes.AsSpan(0, size);
+
+                char[] rentedChars = _charPool.Rent(1000);
+                int cb = Encoding.UTF8.GetChars(basicBytes, rentedChars);
+                Span<char> basicChars = rentedChars.AsSpan(0, cb);
+
+                int separatorIndex = basicChars.IndexOf(':');
+                if (separatorIndex != -1)
+                {
+                    value = new string(basicChars.Slice(separatorIndex + 1));
```

Comment:
I think this will throw an exception if the password side is blank like "`username:`". Consider checking first.

Reply:
Well, it will return an empty string.  Because if you slice off all the characters there are, you end up with an empty span.  it will also, impressively, be a _new_ empty string.  If the string is ":", index will be "0", so we'll slice(1), giving us an empty span (technically it's empty and positioned at the end of the other span, but that's basically impossible to tell)

But it definitely won't be a valid token, which we check later.